modules/images/webp-uploads/load.php

```diff
@@ -573,3 +575,43 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
 	return webp_uploads_img_tag_update_mime_type( $html, 'post_thumbnail_html', $attachment_id );
 }
 add_filter( 'post_thumbnail_html', 'webp_uploads_update_featured_image', 10, 3 );
+
+/**
+ * Returns an array of image sizes that can have alternative mime types. By default, only core sizes
+ * are allowed and will be returned.
+ *
+ * If a custom size wants to have additional mime types support, they will need to have
+ * the provide_additional_mime_types property in the size details or it should be added using the
+ * webp_uploads_image_sizes_with_additional_mime_type_support filter.
+ *
+ * @since n.e.x.t
+ *
+ * @return array An array of image sizes that can have additional mime types.
+ */
+function webp_uploads_get_image_sizes_with_additional_mime_type_support() {
```

Comment:
See my other comment: If we change this to return _all_ sizes with a boolean for their additional MIME type support, we should probably rename this to reflect that data format better, e.g.
```suggestion
function webp_uploads_get_image_sizes_additional_mime_type_support() {
```

Just because then this value is no longer _only_ the sizes _with_ additional MIME type support, I guess.

Reply:
Renamed.